components/gallery/GalleryItemTabsPanel/GalleryItemActivity.vue

```diff
@@ -66,6 +66,13 @@ const cssActive = (value) => {
     ),
   }
 }
+
+const filteredInteractions = computed(() => {
+  if (interactions.value.length === allInteractions.length) {
+    return interactions.value
+  }
+  return interactions.value.filter((interaction) => interaction !== 'UNLIST')
+})
```

Comment:
why not just remove `UNLIST` from `allInteractions`
![CleanShot 2024-04-01 at 17 07 52@2x](https://github.com/kodadot/nft-gallery/assets/44554284/a86b7162-c0db-4b31-bb3e-b5ae833f7e7d)

Reply:
From what I can see we are just adding more code when we could just remove UNLIST from that array

if it's needed once again we can add it back, it's not like it's a lot of code , shouldn't be an issue